app/ui/src/main/java/com/fsck/k9/ui/helper/MaterialColors.java

```diff
@@ -0,0 +1,37 @@
+package com.fsck.k9.ui.helper;
+
+public class MaterialColors {
```

Comment:
I'd prefer for this class to be in Kotlin. The downside is you'll have to write colors like this: `0xFFFFEBEE.toInt()`. But together with the suggested change below I think it's not that bad.